studio/components/ui/MultiSelect/index.tsx

```diff
@@ -39,12 +39,12 @@ export default function MultiSelect({
 }: Props) {
   const ref = useRef(null)
 
-  const [selected, setSelected] = useState(value || [])
-  const [searchString, setSearchString] = useState('')
-  const [inputWidth, setInputWidth] = useState(128)
+  const [selected, setSelected] = useState<string[]>(value || [])
+  const [searchString, setSearchString] = useState<string>('')
+  const [inputWidth, setInputWidth] = useState<number>(128)
 
   // Selected is `value` if defined, otherwise use local useState
-  const selectedOptions = (value || selected).filter((option) => option !== 'public')
+  const selectedOptions = value || selected
```

Comment:
not sure why do we need selectedOptions? cos selected useState already use value as the initial